Dockerfile-ubuntu.template

```
@@ -308,6 +305,7 @@ VOLUME $RABBITMQ_DATA_DIR
 ENV LANG=C.UTF-8 LANGUAGE=C.UTF-8 LC_ALL=C.UTF-8
 
 COPY --chown=rabbitmq:rabbitmq 10-defaults.conf /etc/rabbitmq/conf.d/
+COPY --chown=rabbitmq:rabbitmq 20-management_agent.disable_metrics_collector.conf /etc/rabbitmq/conf.d/
```

:eyes:

```suggestion
COPY --chown=rabbitmq:rabbitmq 10-defaults.conf 20-management_agent.disable_metrics_collector.conf /etc/rabbitmq/conf.d/
```

(or we could even go as far as `*.conf`, which IMO is pretty reasonable now that there are two, but don't feel strongly either way)

---

Got it!